pan3d/viewers/preview.py

```diff
@@ -85,9 +86,11 @@ def __init__(self, server=None, local_rendering=None):
         )
         args, _ = self.server.cli.parse_known_args()
         self.local_rendering = local_rendering
-        if args.wasm:
+        if os.environ.get("BINDER_REQUEST") is not None:
+            self.local_rendering = "vtkjs"
```

Comment:
why do you want to use vtkjs instead of wasm for binder?

Reply:
On my first test w/WASM (before I added this commit, I used the `local_rendering` argument), the renderwindow didn't display. Definitely possible I overlooked something, so I'll change to wasm and retest.

Reply:
So "vtkjs" is ok but "wasm" isn't. It looks like part or all of `trame_vtk_local` isn't getting installed, or is installed in the wrong place. I'll paste the js console messages below, but basically Jupyter aborts because it does not find `__trame_vtklocal/js/style.css`. 

```
index-de5ee105.js:1 
        
        
       GET https://hub.binder.opensci.2i2c.cloud/user/kitware-pan3d-irtfxelw/trame-jupyter-server/servers/esgf-viewer/__trame_vtklocal/js/style.css net::ERR_ABORTED 404 (Not Found)
(anonymous) @ index-de5ee105.js:1
mo @ index-de5ee105.js:1
Ce @ index-de5ee105.js:1
So @ index-de5ee105.js:1
Fs @ index-de5ee105.js:23
await in Fs
(anonymous) @ index-de5ee105.js:23
index-de5ee105.js:23 Uncaught (in promise) Event {isTrusted: true, type: 'error', target: link, currentTarget: null, eventPhase: 0, …}
Fs @ index-de5ee105.js:23
await in Fs
(anonymous) @ index-de5ee105.js:23
```

Reply:
Do we want `os.environ.get("BINDER_REQUEST")` to superseed user input for local rendering request?

Reply:
I think I know why wasm could be missing in the install. 

Let's add `vtklocal.initialize(self.server)` somewhere since the widget get conditionally instantiated.